bridges/otelzap/example_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package otelzap_test
+
+import (
+	"os"
+
+	"go.opentelemetry.io/contrib/bridges/otelzap"
+	"go.opentelemetry.io/otel/log/noop"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+func Example() {
+	// Use a working LoggerProvider implementation instead e.g. using go.opentelemetry.io/otel/sdk/log.
+	provider := noop.NewLoggerProvider()
+
+	// Initialize a zap logger with the otelzap bridge core.
+	// This method actually doesn't log anything on your STDOUT, as everything
+	// is shipped to a configured otel endpoint.
+	// nolint: ineffassign
+	logger := zap.New(otelzap.NewCore("my/pkg/name", otelzap.WithLoggerProvider(provider)))
```

Comment:
```suggestion
	logger := zap.New(otelzap.NewCore("my/pkg/name", otelzap.WithLoggerProvider(provider))) 	// nolint: ineffassign
```

Reply:
`// notlint: inefassign` is not needed anymore due to the split into two distinct examples